### PR TITLE
[IMP] sale,sale_management: improve the UI of the res settings

### PR DESCRIPTION
**Pull request by nikj-odoo**
**Version:**

- master

**Improvement:**

In this PR, aims to enhance the Sale Configuration Settings by removing the default value sections. Additionally, it allows for separate modification of online signature and online payment settings at the first level. Several label modifications have been made, and appropriate descriptive messages have been added for the Online Signature and Online Payment options and add company specific logo in online signature,online payment, and quotation template.

task-3546062

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/sale/wizard/res_config_settings_views.xml [hunk 1]
```diff
@@ -63,45 +63,24 @@
...
-                                    <div class="d-inline-block">days</div>
-                                </div>
+                        <setting  id="online_signature" company_dependent="1" help=" Request customers to sign quotations to validate orders. The default can be changed per order or template." documentation="https://www.odoo.com/documentation/master/applications/sales/sales/send_quotations/get_signature_to_validate.html">
+                            <field name="portal_confirmation_sign"/>
+                        </setting>
+                        <setting id="online_payment" company_dependent="1" help="Request a payment to confirm orders, in full (100%) or partial. The default can be changed per order or template." documentation="https://www.odoo.com/documentation/master/applications/sales/sales/send_quotations/get_paid_to_validate.html">
+                            <field name="portal_confirmation_pay"/>
+                            <div invisible="not portal_confirmation_pay" class="text-nowrap fw-bold">
+                                Payment
+                                <field name="prepayment_percent" widget="percentage" style="width: 3rem"/>
```
Comment (vava-odoo) [inline addons/sale/wizard/res_config_settings_views.xml:73]:
```suggestion
                                <field name="prepayment_percent" widget="percentage" class="oe_inline"/>
```

File: addons/sale/wizard/res_config_settings_views.xml [hunk 2]
```diff
@@ -63,45 +63,24 @@
...
+                            </div>
+                            <div class="mt8" invisible="not portal_confirmation_pay">
+                                <button name='%(payment.action_payment_provider)d' icon="oi-arrow-right" type="action" string="Payment Providers" class="btn-link"/>
+                            </div>
+                        </setting>
+                        <setting id="quotation_validity_days">
+                            <div class="mt16" title="Days between quotation proposal and expiration. 0 days means automatic expiration is disabled">
```
Comment (vava-odoo) [inline addons/sale/wizard/res_config_settings_views.xml:80]: This class doesn't look useful, is it?
```suggestion
                            <div title="Days between quotation proposal and expiration. 0 days means automatic expiration is disabled">
```

File: addons/sale/wizard/res_config_settings_views.xml [hunk 3]
```diff
@@ -63,45 +63,24 @@
...
+                                <label for="quotation_validity_days"/>
+                                <field name="quotation_validity_days" class="text-center" style="width: 5rem;"/>
```
Comment (vava-odoo) [inline addons/sale/wizard/res_config_settings_views.xml:82]: 3 should be more than enough
```suggestion
                                <field name="quotation_validity_days" class="text-center" style="width: 3rem;"/>
```

**Review by vava-odoo — CHANGES_REQUESTED** [review]
Hi @nikj-odoo and @mish-odoo 
Thanks for this work. Please have a look at my comments below.
Also, I think this should be targeting 17.0, not master.
Cheers

**Comment by nikj-odoo** [comment]
Hello @vava-odoo,

I have added the changes you suggested above. Some of the changes were not modified by me, but I still improved the code.

Thanks for the suggestions.

**Review by vava-odoo — APPROVED** [review]

File: addons/sale/wizard/res_config_settings_views.xml [hunk 4]
```diff
@@ -63,45 +63,34 @@
...
+                            </div>
+                        </setting>
+                        <setting id="quotation_validity_days">
+                            <div title="Days between quotation proposal and expiration. 0 days means automatic expiration is disabled">
+                                <label for="quotation_validity_days"/>
+                                <field name="quotation_validity_days" class="text-center" style="width: 3rem;"/>
+                                <div class="d-inline-block">days</div>
+                                <span class="fa fa-lg fa-building-o p-2"
+                                      title="Values set here are company-specific."
+                                      groups="base.group_multi_company"/>
```
Comment (Feyensv) [inline addons/sale/wizard/res_config_settings_views.xml:90]: The company_dependent label was missing for the quotation validity days, I pushed to add this part ;).

**Comment by Feyensv** [comment]
@robodoo r+

**Pull request closed without merging**